js/src/traceable.ts

```diff
@@ -309,6 +338,15 @@ export function traceable<Func extends (...args: any[]) => any>(
     argsConfigPath?: [number] | [number, string];
     __finalTracedIteratorKey?: string;
 
+    /**
+     * Specify which inputs should be treated as attachments and return the remaining arguments.
+     * @param args Arguments of the traced function
+     * @returns A tuple containing the Attachments object and the remaining arguments
+     */
+    getAttachments?: (
```

Comment:
calling this `getAttachments` makes it seem like this should return _just_ the attachments? `separateAttachments` or something similar would be better IMO